doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,1057 @@
+      This feature improves security of the provisioning process.
+
+  * nRF Cloud library:
+
+    * :ref:`lib_nrf_cloud_alert`, which enables applications to generate and transmit messages that comply with the alert features of nRF Cloud.
+    * :ref:`lib_nrf_cloud_log`, which enables applications to generate and transmit messages that comply with the logs features of nRF Cloud.
+
+* Improved:
+
+  * Bluetooth mesh:
+
+    * Updated the default configuration of advertising sets used by the Bluetooth mesh subsystem, which improves performance of the Relay and GATT features with increased throughput, and provides an improved Friend feature to reduce power consumption of low power node.
+    * Mesh light fixture and mesh sensor samples to align with upcoming Bluetooth Networked Lighting Control profiles.
+
+Sign up for the `nRF Connect SDK v2.4.0 webinar`_ to learn more about the new features.
+
+See :ref:`ncs_release_notes_240_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.4.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.4.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.4.0.
+
+:ref:`Toolchain Manager <gs_app_tcm>`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.4.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.4.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.4.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_240_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE and tool support
```

Comment:
Shouldn't we delete sections with the 'no changes yet' note?

Reply:
Removed all such sections except for Bluetooth LE (waiting for input, if any).